code/Model/EditableFormField/EditableFileField.php

```diff
@@ -194,9 +197,23 @@ public function validate()
         return $result;
     }
 
+    public function getFormField()
+    {
+        $field = $this->createFormField();
+        $this->doUpdateFormField($field);
 
+        return $field;
+    }
 
-    public function getFormField()
+    public function getUploadFormField(): FileField
+    {
+        $field = $this->createFormField(true);
+        $this->doUpdateFormField($field);
+
+        return $field;
+    }
+
+    private function createFormField(bool $protectFolder = false): FileField
```

Comment:
I don't think we need the additional `getUploadFormField` and `createFormField` methods?  It seems like they only exist so that we can have an optional param  `bool $protectFolder = false` ... though we only call this once and just set it to true.  The optional param seems redundant?  We shouldn't be encouraging people to have unrestricted access.  And it also goes against what we're doing in `UserDefinedFormAdmin::updateFormSubmissionFolderPermissions()` where we'll just restrict new UDF upload folders created via the CMS UI to Administrator